### PR TITLE
Use new dymola version in ci 

### LINKED ISSUE #1553: Update CI pipeline to latest Dymola
The CI is using dymola 2022, but 2024xRefresh1 is available.

### PR DESCRIPTION
**Pull request by FWuellhorst**
Closes #1553 . Will require the update in some regression statistics, as seen here https://github.com/RWTH-EBC/BESMod/pull/100

### CODE REVIEW

**Comment by FWuellhorst** [comment]
The volume flow in regression results changed slightly in one event: 
image (0.729 instead of 0.728).
The overall plot did not change:
image
Going to update this ref result

File: AixLib/Utilities/Examples/TimeUtilities_test.mo [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
+  AixLib.Utilities.Sources.NightMode nightMode(dayStart = 8, dayEnd = 20) annotation(Placement(transformation(extent = {{-12, -32}, {8, -12}})));
+  AixLib.Utilities.Sources.HourOfDay hourOfDay annotation(Placement(transformation(extent = {{-12, 10}, {8, 30}})));
   Modelica.Blocks.Interfaces.BooleanOutput boolNightMode annotation(Placement(transformation(extent = {{56, -30}, {76, -10}})));
   Modelica.Blocks.Interfaces.RealOutput realSamples[1] annotation(Placement(transformation(extent = {{56, 30}, {76, 50}})));
 equation
   //Connections for real outputs
   realSamples[1] = hourOfDay.HOD;
   //Connection for night mode output
   boolNightMode = nightMode.IsNight.y;
-  connect(boolNightMode, boolNightMode) annotation(Line(points = {{66, -20}, {66, -20}}, color = {255, 0, 255}));
```
Comment (DaJansenGit) [inline AixLib/Utilities/Examples/TimeUtilities_test.mo:14]: This was just nonsense before I guess? :D

**Review by DaJansenGit — APPROVED** [review]
Thanks for the adjustments :)

File: AixLib/Utilities/Examples/TimeUtilities_test.mo (hunk 1 above)
Comment (FWuellhorst) [reply AixLib/Utilities/Examples/TimeUtilities_test.mo:14]: Yes :D The example itself is really weird, as the output is connected to an internal node, `nightMode` does not have an output `isNight`. But I will leave this as is.

**Pull request merged**